ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1339,3 +1339,189 @@
               default-excludes: true
               fingerprint: false
               only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
```

Comment:
I think it's better to put the jobs to public cloud, then we can offer job console output and error log download.